extraction/scraper_vine.py

```diff
@@ -0,0 +1,188 @@
+    terrebonne = 'http://www.terrebonne.lavns.org/roster.aspx'
+    terrebonne = scrape(terrebonne).to_csv(data_file_path('scraped/vine/terrebonne_so_booking.csv'), index=False)
+    washington = 'http://www.washington.lavns.org/roster.aspx'
+    washington = scrape(washington).to_csv(data_file_path('scraped/vine/washington_so_booking.csv'), index=False)
+    webster = 'http://www.webster.lavns.org/roster.aspx'
+    webster = scrape(webster).to_csv(data_file_path('scraped/vine/webster_so_booking.csv'), index=False)
+    wbatonrouge = 'http://www.westbatonrouge.lavns.org/roster.aspx'
+    wbatonrouge = scrape(wbatonrouge).to_csv(data_file_path('scraped/vine/wbatonrouge_so_booking.csv'), index=False)
+    westcarroll = 'http://www.westcarroll.lavns.org/roster.aspx'
+    westcarroll = scrape(westcarroll).to_csv(data_file_path('scraped/vine/westcarroll_so_booking.csv'), index=False)
+    wfeliciana = 'http://www.westfeliciana.lavns.org/roster.aspx'
+    wfeliciana = scrape(wfeliciana).to_csv(data_file_path('scraped/vine/westfeliciana_so_booking.csv'), index=False)
+    winn = 'http://www.winn.lavns.org/roster.aspx'
+    winn = scrape(winn).to_csv(data_file_path('scraped/vine/winn_pd_booking.csv'), index=False)
+
+    #  police departments:
+    bogalusa = 'http://www.bogalusa.lavns.org/roster.aspx'
+    bogalusa = scrape(bogalusa).to_csv(data_file_path('scraped/vine/bogalusa_pd_booking.csv'), index=False)
+    bossiercity = 'http://www.bossierpd.lavns.org/roster.aspx'
+    bossiercity = scrape(bossiercity).to_csv(data_file_path('scraped/vine/bossiercity_pd_booking.csv'), index=False)
+    hammond = 'http://www.hammond.lavns.org/roster.aspx'
+    hammond = scrape(hammond).to_csv(data_file_path('scraped/vine/hammond_pd_booking.csv'), index=False)
+    kinder = 'http://www.kinder.lavns.org/roster.aspx'
+    kinder = scrape(kinder).to_csv(data_file_path('scraped/vine/kinder_pd_booking.csv'), index=False)
+    leesville = 'http://www.leesville.lavns.org/roster.aspx'
+    leesville = scrape(leesville).to_csv(data_file_path('scraped/vine/leesville_pd_booking.csv'), index=False)
+    oakdale = 'http://www.oakdale.lavns.org/roster.aspx'
+    oakdale = scrape(oakdale).to_csv(data_file_path('scraped/vine/oakdale_pd_booking.csv'), index=False)
+    shreveport = 'http://www.shreveport.lavns.org/roster.aspx'
+    shreveport = scrape(shreveport).to_csv(data_file_path('scraped/vine/shreveport_pd_booking.csv'), index=False)
+    sulphur = 'http://www.sulphur.lavns.org/roster.aspx'
+    sulphur = scrape(sulphur).to_csv(data_file_path('scraped/vine/sulphur_pd_booking.csv'), index=False)
+    villeplatte = 'http://www.villeplatte.lavns.org/roster.aspx'
+    villeplatte = scrape(villeplatte).to_csv(data_file_path('scraped/vine/villeplatte_pd_booking.csv'), index=False)
+    winnfield = 'http://www.winnfield.lavns.org/roster.aspx'
+    winnfield = scrape(winnfield).to_csv(data_file_path('scraped/vine/winnfield_pd_booking.csv'), index=False)
```

Comment:
It seems that this part could use a mapping between agency name and URL to scrape. If you're not afraid of diving into `make` config, you can extract this mapping into a json file (i.e. `raw_datasets.json`) for ease of maintenance. But before worrying about it, I recommend replacing Selenium+BeautifulSoup with Scrapy first.

Reply:
I'll look into it. Thanks.

Reply:
Actually, don't worry about make config. Now that I think more about it, putting the links in a simple dictionary is good enough.